SAS_Help_Data/SAS_Help_data_steps.sas

```diff
@@ -17,4 +17,12 @@ proc print data=sashelp.BirthWgt (obs=5);
 run;
 
 
+proc freq data=sashelp.BirthWgt;
+tables race / list missing;
+run;
+
+proc freq data=sashelp.BirthWgt;
+tables agegroup / list missing;
```

Comment:
@Luckydogcoder5 
Hey part of the code is generating errors. You might misspelled the variable